pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -120,14 +120,14 @@ let
          null)
     else "";
 
-  parseResponseFile = if buildPackages.stdenv.cc or null != null && buildPackages.stdenv.cc != "/dev/null"
+  expand-response-params = if buildPackages.stdenv.cc or null != null && buildPackages.stdenv.cc != "/dev/null"
   then buildPackages.stdenv.mkDerivation {
-    name = "parse-response-file";
-    src = ./parseResponseFile.c;
+    name = "expand-response-params";
+    src = ./expand-response-params.c;
     buildCommand = ''
       # Make sure the output file doesn't refer to the input nix path
-      cp "$src" parseResponseFile.c
-      "$CC" -O3 -o "$out" parseResponseFile.c
+      cp "$src" expand-response-params.c
+      "$CC" -O3 -o "$out" expand-response-params.c
```

Comment:
`src` was copied for my version because otherwise there would be a spurious reference in the binary to the source, which would be picked up by Nix.  I'm pretty sure that's because of my use of `assert`, which probably uses `__FILE__` as @Ericson2314 mentioned.  If it doesn't cause trouble with yours, I see no reason to keep the `cp` around.